content/docs/2.0.0/iofogctl/platform-yaml-spec.md

```diff
@@ -4,7 +4,7 @@
 
 ### Note for those upgrading from a previous version
 
-We have updated the yml spec significantly, and while these are small changes in practice, your current yml files will fail to deploy on 1.3.0
+We have updated the yml spec significantly, and while these are small changes in practice, your current yml files will fail to deploy on 2.0.0
```

Comment:
I think we should refer to the release as 2.0

Reply:
updated